python/packages/ai/teams/app.py

```diff
@@ -191,6 +191,32 @@ def meetings(self) -> Meetings[StateT]:
         """
         return self._meetings
 
+    async def get_token_or_sign_in_user(
```

Comment:
where does this method end up getting called?

Reply:
could we also add a few quick test cases for this

Reply:
in the JS package implementation the only reference to "getTokenOrStartSignIn()" occurs in application.ts

then it is seen regularly in bot.ts [example](https://github.com/BMS-geodev/teams-ai/blob/5a6d5f1f04a31d42c0b953ce19d4b6df3f868dd6/js/samples/04.ai-apps/f.whoBot/src/bot.ts#L85)

Reply:
So the first place I would think of this would be in the yet-to-be-worked-on 06.auth.teamsSSO.bot sample? 

Then eventually as an entrypoint anywhere an SSO token is needed?

Reply:
writing tests now